runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsPlugin.java

```diff
@@ -167,6 +176,58 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 
 		switch (eventName)
 		{
+			case "geTagsActive":
+				// tell the script if it should use callbacks when searching and iterating items
+				String s = client.getVar(VarClientStr.INPUT_TEXT).toLowerCase();
+				intStack[intStackSize - 1] = s.startsWith(TAG_SEARCH) ? 1 : 0;
+				break;
+			case "searchGeTagItem":
+				String geSearch = stringStack[stringStackSize - 1];
+				geResults.clear();
+				geIdx = 0;
+
+				String tag = geSearch.replace(TAG_SEARCH, "");
+				List<Integer> tags = tagManager.getItemsForTag(tag);
+				boolean inMembersWorld = client.getWorldType().contains(WorldType.MEMBERS);
+
+				// build stream of item variations if variation tag is present
+				Stream<Integer> itemMappings = tags.stream()
+					.filter(i -> i < 0)
```

Comment:
As i said in previous comment in item mappings, you can just try to convert all the items to itemvariationMapping collection and then do distinct and the tradeable/members filter